stdlib/2and3/ctypes/__init__.pyi

```diff
@@ -271,4 +271,7 @@ class Array(Generic[_T], Sized, _CData):
     def __setitem__(self, i: int, o: Any) -> None: ...
     @overload
     def __setitem__(self, s: slice, o: Iterable[Any]) -> None: ...
-    def __iter__(self) -> Iterable[Any]: ...
+    def __iter__(self) -> Iterator[Any]: ...
+    # Can't inherit from Sized because the metaclass conflict between
```

Comment:
And it is not actually needed because `Sized` is a protocol. I proposed to remove it in https://github.com/python/typeshed/issues/1983 but it looks like it was never done.